### PR TITLE
[full-ci] feat: fetch file info in the sidebar

### PR DESCRIPTION
**Pull request by LukasHirt**
If a single item is selected, fetch a fileInfo. This gives us information about the item even in shared lists.

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18281/24/1
The following scenarios passed on retry:
- webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:114

**Comment by ownclouders** [comment]
Results for <strong>oC10Trashbin</strong> https://drone.owncloud.com/owncloud/web/18281/36/1
The following scenarios passed on retry:
- webUITrashbinRestore/trashbinRestore.feature:222

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsers</strong> https://drone.owncloud.com/owncloud/web/18289/25/1
The following scenarios passed on retry:
- webUISharingInternalUsersShareWithPage/shareWithUsers.feature:91

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18762/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18765/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18766/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:98

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18766/24/1
The following scenarios passed on retry:
- webUISharingInternalGroupsToRootSharingIndicator/shareWithGroups.feature:104

File: packages/web-app-files/src/components/SideBar/Details/FileDetails.vue
Comment (fschade) [inline packages/web-app-files/src/components/SideBar/Details/FileDetails.vue:117]: Dev leftover?

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18857/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersSharingIndicator/shareWithUsers.feature:154
- webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:98

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternalUsers2</strong> https://drone.owncloud.com/owncloud/web/18869/56/1
The following scenarios passed on retry:
- webUISharingInternalUsersShareWithPage/shareWithUsers.feature:135

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait1</strong> https://drone.owncloud.com/owncloud/web/18869/42/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:59

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18874/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersSharingIndicator/shareWithUsers.feature:154

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18874/24/1
The following scenarios passed on retry:
- webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:114

File: packages/web-app-files/src/components/SideBar/SideBar.vue
Comment (kulmann) [inline packages/web-app-files/src/components/SideBar/SideBar.vue:190]: We have a `thumbnail` (small resolution) and `preview` (bigger resolution) for files. The thumbnails are shown in the files list. The previews are shown in the right sidebar. So this should be about previews, not thumbnails, since you're targeting the right sidebar.

**Review by kulmann — CHANGES_REQUESTED** [review]
Getting quite some errors in the js console, see screenshot. 👀 

Screenshot 2021-09-07 at 09 57 06

File: packages/web-app-files/src/components/SideBar/SideBar.vue
Comment (LukasHirt) [reply packages/web-app-files/src/components/SideBar/SideBar.vue:190]: The preview is being loaded in the FileDetails component. In there, it first checks if the file has the thumbnail so that it knows if it should try to load the bigger one or not. I got a bit confused in this but didn't want to bloat the size of this PR even more with refactoring this so I've just made sure the current behaviour is still working.

https://github.com/owncloud/web/blob/8fa0cf0323f59177fa493f4df7dd04de4dc62328/packages/web-app-files/src/components/SideBar/Details/FileDetails.vue#L5

https://github.com/owncloud/web/blob/8fa0cf0323f59177fa493f4df7dd04de4dc62328/packages/web-app-files/src/components/SideBar/Details/FileDetails.vue#L219

Comment (kulmann) [reply packages/web-app-files/src/components/SideBar/SideBar.vue:190]: Argh. We have an option for disabling thumbnails entirely. If that's the case we apparently kill the preview in the sidebar, just because thumbnails in the list are off.
Understandable though that fixing that is out of scope for this PR. 😞

**Comment by kulmann** [comment]
> Getting quite some errors in the js console, see screenshot. 👀
> 
> Screenshot 2021-09-07 at 09 57 06

I still get these errors when having the right sidebar open and navigating through folders (up/down the tree).

**Comment by LukasHirt** [comment]
> I still get these errors when having the right sidebar open and navigating through folders (up/down the tree).

I cannot reproduce this anymore. Switching selected files/folders like crazy, trying multiple selection, no selection... and no error :shrug:

**Comment by kulmann** [comment]
> > I still get these errors when having the right sidebar open and navigating through folders (up/down the tree).
> 
> I cannot reproduce this anymore. Switching selected files/folders like crazy, trying multiple selection, no selection... and no error 🤷

I still can reproduce it the same way as before:
1. check out your branch
2. run `yarn install`
3. run `yarn build:w`
4. run `docker compose up oc10`
5. open dev tools, network tab
6. visit `host.docker.internal:8080` and go to the new design
7. check in the network tab that the web-app-files gets fetched from remote
8. open right sidebar (doesn't matter if in root or in any folder)
9. clear browser console
10. navigate into a folder while the right sidebar is open
11. BAM errors from the screenshot
12. clear browser console
13. navigate into a parent folder via the breadcrumbs
14. BAM again errors from the screenshot

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18957/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:98

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18959/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Review by kulmann — APPROVED** [review]
LGTM 🤩

**Comment by pascalwengerter** [comment]
Please don't merge until #5770 is merged 😬

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18997/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersSharingIndicator/shareWithUsers.feature:154

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPublicManagement</strong> https://drone.owncloud.com/owncloud/web/18997/34/1
The following scenarios passed on retry:
- webUISharingPublicManagement/publicLinkIndicator.feature:27

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternalRoot</strong> https://drone.owncloud.com/owncloud/web/18997/40/1
The following scenarios passed on retry:
- webUISharingExternalToRoot/federationSharing.feature:111

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/19002/24/1
The following scenarios passed on retry:
- webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:114

**Comment by ownclouders** [comment]
Results for <strong>oCISFiles1</strong> https://drone.owncloud.com/owncloud/web/19002/52/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:308

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/19004/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersSharingIndicator/shareWithUsers.feature:154

**Pull request merged**
